docs/patterns/jquery.rst

```diff
@@ -162,5 +162,6 @@ explanation of the little bit of code above:
    argument.  Note that we can use the `$SCRIPT_ROOT` variable here that
    we set earlier.
 
-If you don't get the whole picture, download the :gh:`sourcecode
-for this example <examples/jqueryexample>`.
+Check out the :gh:`example source <examples/javascript>` for a full
+application demonstrating the jQuery on this page, as well as the same
```

Comment:
"the jQuery" reminds me of "hello sir pls help me with my jquery it shows syntax error" SO questions from people who don't know that jQuery is just JS in the end.

What about making the `fetch` one the default example? All modern browsers (IE11 isn't really modern) support it and there are good polyfills/ponyfills available for those who do need to support IE.

And you even use a polyfill below (`whatwg-fetch`) so it's going to work everywhere.

Reply:
I do want to go through the docs and rewrite them at some point, but I just didn't have time now. Running the app shows the jQuery demo last of the three, for what it's worth.